get_hrc.py

```diff
@@ -18,46 +18,65 @@
                     help='Directory for output data files')
 args = parser.parse_args()
 
-url = 'ftp://ftp.swpc.noaa.gov/pub/lists/pchan/Gp_pchan_5m.txt'
-colnames = ('year month dom  hhmm  mjd secs p1  p2  p3 '
-            'p4  p5  p6  p7  p8  p9 p10 p11').split()
+url = 'https://services.swpc.noaa.gov/json/goes/primary/differential-protons-6-hour.json'
 
 last_err = None
 for _ in range(3):
     try:
-        urlob = urllib.request.urlopen(url)
-        urldat = urlob.read().decode()
+        json_file = requests.get(url)
+        data = json_file.json()
         break
     except Exception as err:
         last_err = err
         time.sleep(5)
 else:
-    print('Warning: failed to open URL {}: {}'.format(url, last_err))
+    print(f'Warning: failed to open URL {url}: {last_err}')
     sys.exit(0)
 
-dat = ascii.read(urldat, Reader=ascii.NoHeader, names=colnames,
-                 data_start=3)
-hrc_bad = (dat['p4'] < 0) | (dat['p5'] < 0) | (dat['p6'] < 0)
+dat = Table(data)
 
-mjd = dat['mjd'] + dat['secs'] / 86400.
+secs = []
+for row in dat:
+    sec = DateTime(row['time_tag'][:-1], format='fits').secs
+    secs.append(sec)
+dat['time'] = secs
 
-secs = DateTime(mjd, format='mjd').secs
+descrs = [('time', 'f8'), ('hrc_shield', 'f8')]
+channels = ['P1', 'P2A', 'P2B', 'P3', 'P4', 'P5', 'P6',
+            'P7', 'P8A', 'P8B', 'P8C', 'P9', 'P10']
 
-hrc_shield = (6000 * dat['p4'] + 270000 * dat['p5']
```

Comment:
Decided that we want to keep the old computation as a comment

Reply:
I see the comments have been updated to describe this, but I think we really just want this bit of code in a comment.

Reply:
Added pre-GOES-16 computations as comments.